common/include/opae/cxx/core/shared_buffer.h

```diff
@@ -88,9 +88,17 @@ class shared_buffer {
 
   /** Retrieve the virtual address of the buffer base.
    *
-   *  @note assiging this to a variable should be done in limited scopes
-   *  as the variable can outlive the shared_buffer object creating the danger
-   *  of a dangling pointer
+   *  @note Instances of a shared buffer can only be created using either
```

Comment:
maybe an example to clarify/drive home the point?

Reply:
Example here? The cxx hello fpga isn't enough?

Reply:
Maybe offer up the issue submitter's code as an example of de-referencing the invalid pointer after the scope of the shared_ptr..  Having seen the code, it was clear to me what you were describing.  I'm concerned that others might not get the point by only reading the prose.